spec/requests/api/products_spec.rb

```diff
@@ -0,0 +1,10 @@
+RSpec.describe Api::ProductsController, type: :request do
+  describe 'GET /product' do
+    before do
```

Comment:
Please consider this:

```ruby
let!(:products) { 3.times { FactoryBot.create(:product) } }
```

Then, you can test if the response has data about 3 products.